src/shapes/image.class.ts

```diff
@@ -526,7 +527,7 @@ import { FabricObject } from './fabricObject.class';
           this._lastScaleY = 1;
         }
         if (!fabric.filterBackend) {
-          fabric.filterBackend = fabric.initFilterBackend();
+          fabric.filterBackend = initFilterBackend();
```

Comment:
tree shaking should relief us from this task?
we can declare it in the file so if it is imported it is included and that's it...
well I guess for the cdn....

Reply:
mmm i think there are cases in which you want to swap the instance.
It was happened here, but probably a setFilterBackend method is all we need in case you want to switch from 2d to webgl on request